xcode/Subconscious/Shared/Components/Detail/UserProfileDetailView.swift

```diff
@@ -350,8 +354,8 @@ struct UserProfileDetailModel: ModelProtocol {
     var address: Slashlink? = nil
     var user: UserProfile? = nil
     
-    var recentEntries: [EntryStub] = []
-    var following: [StoryUser] = []
+    var recentEntries: [EntryStub]? = nil
+    var following: [StoryUser]? = nil
```

Comment:
Why do these properties need to be optional? Loading state?

Suggestion: comment explaining.

Reply:
Added a comment but repeating here:

```
    // These are optional so we can differentiate between 
    // (1) first load (no content, show placeholder state)
    // (2) a refresh (existing content, keep showing old content during load)
```